src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -25,28 +27,14 @@ See the first part of the tutorial.
 * Install Node.js and launch your project locally.
 See the second part of the tutorial.
 
-== Changing a custom value
+== Making a change in the source code
 
-For the https://withkoji.com/~Svarog1389/element-match[Element Match] game, a new game title would be a simple change that is obvious to test.
-To change the title in the Koji editor, you would click *VCC Explorer > Game Settings* on the left side, which opens the Game Settings editor in Visual mode.
-This mode shows the visual controls that will enable non-technical users to customize your template when you have finished developing it.
+Let's make a small change in the https://withkoji.com/~Svarog1389/rxkd[Falling Objects] game, just for testing purposes.
 
-Because Visual mode is not available when you are developing your project locally, you must edit the JSON customization file directly.
+. Open `frontend\src\Components\View\LeaderboardMini.js`.
+Change `<Title>{`Top Players`}</Title>` to `<Title>{`Leaderboard`}</Title>` and save the file.
 
-. Open the `.koji/customization/string.json` in your text editor.
-. Change the title and save the file.
-+
-[source,json]
-----
-  "strings": {
-    "fontFamily": "https://fonts.googleapis.com/css?family=Lilita+One",
-    "title": "CUSTOM TITLE", <1>
-    "instructions1": "Swipe the Elements to move them",
-    ...
-  }
-----
-<1> Change the value for the `title` key.
-. In your browser, go to `\http://0.0.0.0:8080`, and make sure the splash screen shows the new title.
+. In your browser, go to `\http://0.0.0.0:8080`, and make sure the leaderboard shows the new title.
 
 == Moving changes from local to live
```

Comment:
Can you double-check with Marko or Sean to see if you still need to publish from the editor or if pushes to origin trigger a full deployment? I can't remember if we changed that or if I just hoped that we did. You might still need to pull changes into the editor if you want to develop there. If so, update these steps and add optional steps for pulling changes into the editor. If not, just resolve this ticket as wishful thinking.